mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -1332,167 +1482,187 @@ class spawn_point_types {
-			};
-		};
+	class anzac_air_transport_all: parent_air_transport_all {
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_ANZAC;
 	};
-
-	class air_usmc {
-		name = "USMC Helicopters";
+	class anzac_grnd_transport_light {
+		name = "Light Transport (ANZAC)";
 		RESPAWN_SHORT;
+		lockTeams[] = LOCKED_ANZAC;
 		class categories {
-			class ch34_t {
-				name = "CH-34";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { { "ch34","usmc" } };
-				exclude[] = { { "pavn" } };
+			class dirtranger_t {
+				name = "Dirt Ranger";
+				icon = VEHICLE_ICON_CAR;
+				include[] = { { "dirtranger", "transport" } };
+				exclude[] = { {"armed" } };
 			};
-			class uh1 {
-				name = "UH-1";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { { "uh1", "usmc" } };
-				exclude[] = { { "pavn" } };
+			class dirtranger_p {
+				name = "Armed Dirt Ranger";
+				icon = VEHICLE_ICON_CAR;
+				include[] = { { "dirtranger", "armed" } };
+				exclude[] = { {"transport" } };
 			};
 		};
 	};
-
-	class air_tf {
-		name = "SOG Helicopters";
+	class anzac_air_transport_custom {
+		name = "Transport Helicopters (ANZAC)";
 		RESPAWN_SHORT;
+		lockTeams[] = LOCKED_ANZAC;
 		class categories {
 			class cayuse {
 				name = "OH-6A";
 				icon = VEHICLE_ICON_HELO;
-				include[] = { {"oh6a","transport"}};
-				exclude[] = { {"firesupport","scout"} };			
+				include[] = { {"oh6a","transport"} };
+				exclude[] = { {"firesupport","pavn"} };
 			};
 			class uh1 {
 				name = "UH-1";
 				icon = VEHICLE_ICON_HELO;
-				include[] = { { "uh1", "tf" } };
-				exclude[] = { { "pavn" } };
+				include[] = { {"uh1","anzac"} };
+				exclude[] = { {"firesupport","usmc","medical","cia","arvn","usaf","army","pavn","7th","frogmen"} };
 			};
 			class ch47 {
-				name = "CH-47";
+				name = "CH-47A";
 				icon = VEHICLE_ICON_HELO;
-				include[] = { {"ch47","transport","army"} };
-				exclude[] = { {"firesupport","7th"} };
+				include[] = { {"ch47","army"} };
+				exclude[] = { {"firesupport","medical","army","pavn","7th"} };
 			};
 		};
 	};
 
-	class air_dustoff {
-		name = "Dust-off and Medevac";
-		RESPAWN_SHORT;
+	// blackhorse /////////////////////////////////////////////////////////////////
+
+	class blackhorse_air_transport_all: parent_air_transport_all {
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_BLACKHORSE;
+	};
+	class blackhorse_statics_105s {
+		name = "105 Artillery (Blackhorse)";
+		WRECK_LONG;
+		lockTeams[] = LOCKED_BLACKHORSE;
 		class categories {
-			class uh1 {
-				name = "UH-1 Medical";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { { "uh1", "medical", "dustoff" } };
-				exclude[] = { { "pavn" } };
-			};
-			class ch47 {
-				name = "Ch-47 Medical";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { { "ch47", "medical", "dustoff" } };
-				exclude[] = { { "pavn" } };
+			class m101 {
+				name = "M101";
+				icon = VEHICLE_ICON_STATIC;
+				include[] = { { "static", "artillery" } };
+				exclude[] = { {"mortar" } };
 			};
 		};
 	};
-
-	class air_transport_light {
```

Comment:
now `unlocked_air_transport_light`, template for it is `parent_air_transport_light:`